app/main.py

```diff
@@ -35,6 +35,7 @@
 # Main Dash imports, used to instantiate the web-app and create callbacks (ie. to generate interactivity)
 import dash
 from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
```

Comment:
Yes, it's a great tool! I discovered it when looking for callback techniques allowing a given event to be triggered by two different inputs, it would probably be worth using it regularly to avoid weird behaviours with the map